src/shared/filesystemUtilities.ts

```diff
@@ -220,3 +221,24 @@ export async function cloud9Findfile(dir: string, fileName: string): Promise<vsc
     }
     return []
 }
+/**
+ * @returns  A string path to the last locally stored download location. If none, returns the users 'Downloads' directory path.
+ */
+export function getDefaultDownloadPath(): string {
+    const lastUsedPath = globals.context.globalState.get('aws.downloadPath')
+    if (lastUsedPath) {
+        if (typeof lastUsedPath === 'string') {
+            return lastUsedPath
+        }
+        getLogger().warn("Expected stored 'aws.downloadPath' to return a string")
```

Comment:
This is an unusual and suspicious state so error is the correct log-level.

```suggestion
        getLogger().error("Expected stored 'aws.downloadPath' to return a string")
```